### ISSUE TITLE
Improve Documentation Search

### ISSUE THREAD

**Issue by surjithctly** [tailwindlabs/tailwindcss.com#756]
Most of the time, the Algolia search doesn't show what I need. So had to manually look up every time. See for example. This is just an example, but its true for many words. 

![image](https://user-images.githubusercontent.com/1884712/107947064-dfd66480-6fb7-11eb-80bc-45c1a19f8267.png)

**Comment by adamwathan** [comment]
Are you trying to search for "transparent" on the page you are already on? If so you should just be using the "find" feature in the browser, the search is for finding dedicated pages/section for bigger picture ideas.

Either way unfortunately not much we can do to drastically overhaul how Algolia handles the search and this is the first time in 4 years that we've heard anything but positive feedback on it so it's not likely something we are going to work on, sorry 😞

**Comment by surjithctly** [comment]
I opened it in the same page just to illustrate this. But its not the case actually. I think you feed titles and descriptions only to Algolia. usually I search CSS Terms & parts of class names. Which always fails. Also `transparent` word is also an example. 

I'm not opening this by just one search today, But because of the failed search for the past few months. Here's what I found what's happening: when a search term does appear in the top table and does not repeat anywhere in the page, it does not count. 

![image](https://user-images.githubusercontent.com/1884712/107952475-d05b1980-6fbf-11eb-816a-c14cebd58e4a.png)

So, the possible solution is to let the Algolia read Top table as well. I'm not sure how hard this is to implement. 

If its easy, I would suggest to do it. 

Thanks again

~Surjith
